src/components/emptyStatePages/EmptyStateContent.tsx

```diff
@@ -47,7 +47,7 @@ const useStyles = makeStyles((theme: Theme) => ({
   }),
   listItem: {
     flex: '1 1 auto',
-    maxWidth: '200px',
+    maxWidth: '205px',
```

Comment:
With the icon the "Import Inventory" string was just slightly too long to fit in a button with padding and this maxWidth.  Open to suggestions for a different way to handle this.

Reply:
This looks good to me